### PR TITLE
feat: elec system sounds and other huge changes

### PR DESCRIPTION
**Pull request by hotshotp**
## Summary of Changes

I have changed many many sounds in this PR, I will list them here:

- I have changed the avionics fan sound to be more realistic using higher quality recordings from komp
- Changed the engine start sounds to have a more realistic sounding fuel introduction for a cold start (when the crank phase is skipped, it introduces fuel at a lower n2 percentage which i simulated)
- Elongated the engine shutdown sound
- Added the power transfer sounds when switching from buses
- When turning on batteries, there is a clunk sound for one battery and a quieter clunk sound for both batteries when they are on, which is then followed by the light transformer whines and the CIDS chimes - this means that all of the sounds now follow the correct electric bus
- The whines that the elec system makes are all improved too
- The cabin ambience is improved also where the cabin fans now come from around the leading edge of the wing as that is where they are located. Now the light transformers are also heard inside the cabin and there is a small whine that comes from the avionics fan that is heard if you pay attention in the front to mid cabin
- There are now pack and brake fan spool down sounds
- Made the exterior idle be heard from farther away

## Additional context

Known issues:
-When the power up sequence is interrupted, like the avionics ventilation test, the sound will keep playing even when cold and dark, this is due to a wwise limitation of the sim now allowing wwise stop events to play and I can not do anything about this currently

-When the left engine is shut down the aircraft loses and gains power making many sounds at once play - this is not an issue with sounds and with the elec system - this will be fixed by @davidwalschots soon in a future PR

Discord username (if different from GitHub):

## Testing instructions

- Bat 1 and 2 to on and listen to the new clunks and whines
- Start APU and turn off ext pwr to listen to power transfer
- Listen to the new avionics fan sound
- Turn on APU bleed and turn it off - then go to exterior view to listen to the shutdown
- Turn brake fan on and go to exterior view to turn it off to listen to the shutdown 
- Go to front/mid cabin (showcase view 13) and listen to the improved ambience
- Start engine and listen to new fuel intro - listen to exterior for the first eng start and interior for the second eng start
- Check that exterior idle is heard from farther away (1000m instead of 500m)
- Shut down engines and listen from interior

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by wpine215 — APPROVED** [review]

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -226,6 +226,9 @@
 1. [ND] Add VOR/ADF needles on ARC display mode - @tracernz (Mike)
 1. [MCDU] Rework PERF APPR page to Honeywell Rev. 2 standard - @tracernz (Mike)
 1. [EFB] Set font of OFP to monospace - @chkgk (Chriskgk#4359)
+1. [SOUND] Added electrical system sounds - @hotshotp (Boris)
+1. [SOUND] General improvements to cockpit and cabin ambience - @hotshotp (Boris)
+
```
Comment (pepperoni505) [inline .github/CHANGELOG.md:231]: Remove this line
```suggestion
```

**Comment by St54Kevin** [comment]
Category: QA Tester
Name: St54Kevin#3583
Date of testing: 22/03/2021
Version of the sim: 1.14.5.0
PR Tested: #4064  
Tier of testing: 1
Changes to observe: Sounds

**Steps you've made to test the Pull Request:**
1. Loaded C&D and listened to following sounds:
2. BAT1 & BAT2 clunk
3. Avionics sound
4. EXT PWR transfer with APU running and APU bleed shutdown
5. Cabin ambience with bleed 
6. Brake Fan shutoff exterior sound
7. ENG start sound interior and exterior
8. ENG shutdown sound interior

**Media:**

**Issues (If Any):** - 

**Conclusions of the testing:** Amazing as always. I cant say anything about the sounds. They were brilliant before and are even more now. New ENG start sounds are pretty badass, especially from outside. Love it!

**Comment by FinalLightNL** [comment]
Category: Contributor
Name: FinalLight #2113
Date of testing: 03/22/2021
Version of the sim: 1.14.5.0
PR Tested: #4064 
Tier of testing: 1
Changes to observe: Sounds

**Steps you've made to test the Pull Request:**
**1.** Loaded cold&dark
**2.** Listened to the Battery 1 & 2 clunk & whines
**3.** Listened to the avionics
**4.** Listened to the ext power and apu starting up and shutting down including apu bleed
**5.** Started up the engines twice to listen from the interior and exterior
**6.** Listened to the Brake fans shutoff sound.
**7.** Listened to the cabin ambient noise.
**8.** Went to showcase drone and listened to the idle engines from a distance.
**9.** Listened to the engines turning off from the interior.

**Flight notes:** -

**Media:** -

**Issues (If Any):** -

**Overall Rating:** Good
**Conclusions of the testing:** It sounds amazing to me, i thought the sounds were allready great but this is just a cherry on top of the cake.
The engines from the exterior view sounds magnificent, from the wooshing air being sucked into the engines to the engine sounds themselves.
the new avionic fan sound is more pleasing also.
All in all it LGTM!

**Comment by DavidEGTE** [comment]
Category: QA Tester
Name: David#2177
Date of testing: 22/03/2021
Version of the sim: 1.14.5.0
PR Tested: #4064 
Tier of testing: 2 

Changes to observe:
electric system sounds and other huge changes

Testing technique: Try-out

Steps you've made to test the Pull Request:
1. Spawned C&D
2. Followed PR instructions
3. Played with functionality's 

Flight notes (When done a full flight):
N/A
Media: Insert screenshots, recordings, or anything relevant to the review here.
N/A
Issues (If Any): Write any issues
N/A
Steps to achieve the issue:
N/A

Overall Rating: Good 
Conclusions of the testing: GTG

**Comment by ExampleWasTaken** [comment]
Category: QA Trial
Name: ExampleWasTaken#0886
Date of testing: 22/03/2021
Version of the sim: 1.14.5.0
PR Tested: #4064
Tier of testing: 2
Changes to observe:
elec system sounds and other HUGE CHANGES

Testing technique:
1. Listened to the new Bat 1 and 2 clunks and whines
2. Started the APU, turned of EXT PWR and listened to the power transfer
3. Listened to the new avionics fan sound
4. Turned off the APU bleed and listened to the shutdown from the exterior view
5. Turned brake fan on and off and listened to the shutdown sound
6. Listened to the improved ambience with engines shut down and running
7. Checked that the exterior idle is heard from 1000m away
8. Shut down engines and listened to the shutdown and power transfer to APU

Flight notes (When done a full flight): N/A

Media:

Issues (If Any): I don't have a RL reference but I find the bleed shutdown a little bit too fast.
Turn APU bleed off and go to exterior view

Overall Rating: Amazing
Conclusions of the testing: Everything works perfectly. Great work!

**Comment by brunocfalcao** [comment]
The sound of the OVERHEAD INTG LT when you rotate it is no longer hearable, might have been an introduced bug?

This one:
image

**Comment by DavidEGTE** [comment]
> The sound of the OVERHEAD INTEG LT when you rotate it is no longer hearable, might have been an introduced bug?
> 
> This one:
> image

I've backlogged a A32NX before this update and there was no sound even

tried to investigate the issue with other testers, dev team explained:

no, it's not supposed to have a sound
it's a rotary knob with no steps

Glad that this could help you @brunocfalcao

**Comment by brunocfalcao** [comment]
Thanks @DavidEGTE , strangely, on my version I could hear the tick that overhead knob. No worries, thanks for the clarification!

**Pull request merged**